azure-sql/database/connect-query-java.md

```diff
@@ -126,28 +179,74 @@ Using your favorite IDE, create a new Java project, and add a `pom.xml` file in
     <name>demo</name>
 
     <properties>
-        <java.version>1.8</java.version>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <java.version>11</java.version>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
     </properties>
 
     <dependencies>
         <dependency>
             <groupId>com.microsoft.sqlserver</groupId>
             <artifactId>mssql-jdbc</artifactId>
-            <version>7.4.1.jre8</version>
+            <version>12.2.0.jre11</version>
         </dependency>
+       <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-identity</artifactId>
```

Comment:
I don't think you'll need a dependency on azure-identity. The JDBC driver should give you what you need. See my other comment about using the DefaultAzureCredential support.